tox.ini

```diff
@@ -1,7 +1,11 @@
 [tox]
-envlist = py{27}-six{19,Latest}
+envlist = py26,py27-six19,py27-sixLatest,py33,py34,py35,flake8
```

Comment:
I'd prefer changes to the tox file to occur in a separate PR (and I've got a lot of this in #417 already 😃 )